foundations/foundation-vertx/src/main/java/org/apache/servicecomb/foundation/vertx/ClientEvent.java

```diff
@@ -20,21 +20,35 @@
 /**
  * Notice: this event will raised in vertx eventloop thread, so do not run any block code
  */
-public class ClientClosedEvent {
+public class ClientEvent {
   private final String address;
 
+  private final EventType eventType;
+
+  private final ServerType serverType;
+
   private final int totalConnectedCount;
 
   public String getAddress() {
     return address;
   }
 
+  public EventType getEventType() {
+    return eventType;
+  }
+
+  public ServerType getServerType() {
+    return serverType;
+  }
+
   public int getTotalConnectedCount() {
     return totalConnectedCount;
   }
 
-  public ClientClosedEvent(String address, int totalConnectedCount) {
+  public ClientEvent(String address, EventType eventType, ServerType serverType, int totalConnectedCount) {
```

Comment:
I'd prefer ClientEvent -> ConnectionEvent & ServerType -> TransportType, how do you think?

Reply:
that's great,thanks, done.